plugins/woocommerce/src/Blocks/BlockPatterns.php

```diff
@@ -205,16 +205,24 @@ public function register_ptk_patterns() {
 			return;
 		}
 
-		// The most efficient way to check for an existing action is to use `as_has_scheduled_action`, but in unusual
-		// cases where another plugin has loaded a very old version of Action Scheduler, it may not be available to us.
-		$has_scheduled_action = function_exists( 'as_has_scheduled_action' ) ? 'as_has_scheduled_action' : 'as_next_scheduled_action';
```

Comment:
Don't we want to keep this fallback logic? 

We have in few places:

https://github.com/woocommerce/woocommerce/blob/48eb0030dce131947367d632b8dfecedb2d71def/plugins/woocommerce/src/Internal/BatchProcessing/BatchProcessingController.php#L552-L553